docker-compose.yml

```diff
@@ -842,9 +842,14 @@ services:
       PYTEST_ARGS:  # inherit
     volumes: *conda-volumes
     command: &python-conda-command
+    # pytest-cython needs an upper pin of 0.3.0 due
+    # to an issue with typing annotiations
+    # https://github.com/lgpage/pytest-cython/issues/67
+    # With 0.3.* bug fix release, the pin can be removed
```

Comment:
Is it worth creating a GH issue to unpin and add the issue # in the comment here?

Reply:
Yes, will do!

Reply:
Done: https://github.com/apache/arrow/issues/41002